CMSgov/design-system
# [WNMGDS-611] Generically copy src files

### Summary
- Files that arent located in `components`, `styles`, `fonts`, and `images` in `src` will automatically be copied to `dist`

### How to test
- Add files and folders under `src`, make sure they are in dist after running `yarn start`
- Add files that wont be processed by the build scripts (i.e. `css` files in `components`, `js` files in `styles`), make sure they arent in dist after running `yarn start`
